Neos.ContentRepository.Core/Classes/Subscription/Engine/SubscriptionEngine.php

```diff
@@ -25,7 +28,7 @@
 use Neos\ContentRepository\Core\Subscription\Subscriptions;
 
 /**
- * @api
+ * @internal public API is the {@see ContentRepository::handle()} and the {@see ContentRepositoryMaintainer}
  */
 final class SubscriptionEngine
```

Comment:
the `ContentRepositoryMaintainer` allows us to make the subscription engine internal which is important to be able to extend it and to not expose too many concepts at first.